### PR TITLE
Update Option Tabs

### PR DESCRIPTION
**Pull request by ismailsunni**
### What does it fix?
* Ticket: #4482 
* Funded by: DFAT
* Description: 
   - First step to have InaSAFE profile
   - Reorder tab in the InaSAFE options and make it prettier

update_logo

### Checklist:
- [x] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [x] Request someone to review or test your PR

### CODE REVIEW

File: safe/gui/tools/options_dialog.py [hunk 1]
```diff
@@ -160,6 +162,7 @@ def __init__(self, iface, parent=None, qsetting=''):
         # hide custom template dir toggle
         self.custom_templates_dir_checkbox.hide()
         self.splitter_custom_report.hide()
+        # self.set_organisation_logo()
```
Comment (Gustry) [inline safe/gui/tools/options_dialog.py]: You keep it for legacy or is-it really old code?

File: safe/gui/tools/options_dialog.py [hunk 2]
```diff
@@ -412,7 +418,7 @@ def on_toolNorthArrowPath_clicked(self):
 
     # noinspection PyPep8Naming
     @pyqtSignature('')  # prevents actions being handled twice
```
Comment (Gustry) [inline safe/gui/tools/options_dialog.py]: Can'we remove these decorator too? I guess if we use explicit connect, it's fine, no? It's only a question.

File: safe/gui/tools/options_dialog.py [hunk 3]
```diff
@@ -412,7 +418,7 @@ def on_toolNorthArrowPath_clicked(self):
...
-    def on_toolOrganisationLogoPath_clicked(self):
+    def on_open_organisation_logo_path_button_clicked(self):
         """Auto-connect slot activated when logo file tool button is clicked.
```
Comment (Gustry) [inline safe/gui/tools/options_dialog.py]: Can we remove these autoconnects while you are working on this file?

File: safe/gui/tools/options_dialog.py (hunk 1 above)
Comment (ismailsunni) [reply safe/gui/tools/options_dialog.py]: Just forget to delete :)

File: safe/gui/tools/options_dialog.py (hunk 2 above)
Comment (ismailsunni) [reply safe/gui/tools/options_dialog.py]: I think we can. I never use this decorator actually.

File: safe/gui/tools/options_dialog.py (hunk 3 above)
Comment (ismailsunni) [reply safe/gui/tools/options_dialog.py]: Ok, I will. Thanks for the review.

**Pull request merged**